examples/gradio/service.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations  # I001
+
+import gradio as gr
+
+import bentoml
+
+with bentoml.importing():
+    import torch
+    from transformers import pipeline
+
+EXAMPLE_INPUT = "Breaking News: In an astonishing turn of events, the small \
+town of Willow Creek has been taken by storm as local resident Jerry Thompson's cat, \
+Whiskers, performed what witnesses are calling a 'miraculous and gravity-defying leap.' \
+Eyewitnesses report that Whiskers, an otherwise unremarkable tabby cat, jumped \
+a record-breaking 20 feet into the air to catch a fly. The event, which took \
+place in Thompson's backyard, is now being investigated by scientists for potential \
+breaches in the laws of physics. Local authorities are considering a town festival \
+to celebrate what is being hailed as 'The Leap of the Century."
+
+
+def summarize_text(text: str) -> str:
+    svc_instance = bentoml.get_current_service()
+    return svc_instance.summarize([text])[0]
+
+
+io = gr.Interface(
+    fn=summarize_text,
+    inputs=[gr.Textbox(lines=5, label="Enter Text", value=EXAMPLE_INPUT)],
+    outputs=[gr.Textbox(label="Summary Text")],
+    title="Summarization",
+    description="Enter text to get summarized text.",
+)
```

Comment:
I wonder can we automatically generate this from the schema?
Or do we want to give more flexibility to users?

Reply:
Isn't you need to have this interface to mount it to a bentoml service?

edit: I just realized that the fn called the service instance below :)

Reply:
we should give more flexibility to users. the gradio UI can be quite dynamic and includes UI specific logic, no need to couple it with the model inference / pipeline logic defined in BentoML APIs.